test/fixtures/steps.js

```diff
@@ -51,6 +51,14 @@ module.exports = {
       section: 'enquiry-details'
     }
   },
+  '/additional-2': {
+    fields: [
+      'additions-names'
```

Comment:
I don't understand the test case for this? Is additions names used elsewhere?

This seems like a large breaking change...?